android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -578,6 +578,7 @@ public void submitTransaction(@NonNull Transaction transaction)
             Util.logException(TAG, invalidTransactionException);
             throw invalidTransactionException;
         }
+        return txResponse.getBlockCount();
```

Comment:
```suggestion
        return txResponse.getBlockCount() - 1;
```
This returns block count. To make this the same as iOS, we should return the index (count - 1)